classes/task/send_files.php

```diff
@@ -36,7 +36,9 @@ public function plagiarism_vericite_send_files() {
 
             foreach ($dbfiles as $dbfile) {
                 try {
-                    $customdata = unserialize(base64_decode($dbfile->data));
+                    //Get data for this row
+                    $row = $DB->get_record('plagiarism_vericite_files', array('id' => $dbfile->id), 'id, cm, userid, identifier, data, status, attempts');
```

Comment:
Shouldn't the first DB query be only getting the IDs? (i.e. line 26 $DB->get_records)